### PR TITLE
RecordedFuture added outputs to the commands

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/31375

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by moishce** [comment]
@Arsenikr Can you please help me here with the missing descriptions?

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 1]
```diff
@@ -115,20 +129,51 @@ script:
...
-  - name: ip
-    arguments:
-    - name: ip
-      required: true
-      default: true
+    - contextPath: Domain.Tags
+      description: Tags that are associated with the Domain
+      type: String
+    - contextPath: Domain.CommunityNotes.Note
+      description: '...'
```
Comment (ytauman) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:136]: A summary note on the mentioning of the domain in a source monitored by Recorded Future.

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 2]
```diff
@@ -115,20 +129,51 @@ script:
...
+      type: String
+    - contextPath: Domain.CommunityNotes.Timestamp
+      description: '...'
```
Comment (ytauman) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:139]: The timestamp in which the community note was created.

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 3]
```diff
@@ -115,20 +129,51 @@ script:
...
+      type: Date
+    - contextPath: Domain.Publications.Source
+      description: '...'
```
Comment (ytauman) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:142]: The publication's source.

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 4]
```diff
@@ -115,20 +129,51 @@ script:
...
+      type: String
+    - contextPath: Domain.Publications.Title
+      description: '...'
```
Comment (ytauman) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:145]: The title of the published information.

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 5]
```diff
@@ -115,20 +129,51 @@ script:
...
+      type: String
+    - contextPath: Domain.Publications.Link
+      description: '...'
```
Comment (ytauman) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:148]: A link to the published information.

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 6]
```diff
@@ -115,20 +129,51 @@ script:
...
+      type: String
+    - contextPath: Domain.Publications.Timestamp
+      description: '...'
```
Comment (ytauman) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:151]: The timestamp in which the information was published.

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 7]
```diff
@@ -160,20 +206,51 @@ script:
...
       type: date
-    description: Returns threat intelligence information for an IP address in Recorded Future.
-  - name: file
-    arguments:
-    - name: file
+    - contextPath: IP.Tags
+      description: Tags that are associated with the Domain
+      type: String
+    - contextPath: IP.CommunityNotes.Note
+      description: '...'
```
Comment (ytauman) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:213]: A summary note on the mentioning of the IP in a source monitored by Recorded Future.

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 8]
```diff
@@ -220,23 +297,56 @@ script:
...
-  - name: recorded-future-get-related-entities
-    arguments:
-    - name: entityType
-      required: true
-      auto: PREDEFINED
+    - contextPath: File.Tags
+      description: Tags that are associated with the Domain
+      type: String
+    - contextPath: File.CommunityNotes.Note
+      description: '...'
```
Comment (ytauman) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:304]: A summary note on the mentioning of the file in a source monitored by Recorded Future.

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 9]
```diff
@@ -530,27 +654,67 @@ script:
...
-    description: Returns threat intelligence information for a URL in Recorded Future.
-  - name: recorded-future-get-threats-hash
-    arguments:
-    - name: detailed
-      auto: PREDEFINED
+    - contextPath: URL.Tags
+      description: Tags that are associated with the Domain
+      type: String
+    - contextPath: URL.CommunityNotes.Note
+      description: '...'
```
Comment (ytauman) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:661]: A summary note on the mentioning of the URL in a source monitored by Recorded Future.

File: Packs/Recorded_Future/ReleaseNotes/1_0_3.md [hunk 10]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Recorded Future
+- Added outputs to the **IP**, **URL**, **File**, **Domain** commands.
```
Comment (ShahafBenYakir) [inline Packs/Recorded_Future/ReleaseNotes/1_0_3.md:4]:
```suggestion
- Added new outputs to the **IP**, **URL**, **File** and **Domain** commands.
```

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 11]
```diff
@@ -1,95 +1,109 @@
+category: Data Enrichment & Threat Intelligence
```
Comment (ShahafBenYakir) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:1]: I see a lot by I believe that most of the things here are not really changes. Right? Can you summarize the changes made here in another way?

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 12]
```diff
@@ -115,20 +129,51 @@ script:
...
+      type: String
+    - contextPath: Domain.Publications.title
+      description: The title of the published information.
+      type: String
+    - contextPath: Domain.Publications.link
+      description: A link to the published information.
+      type: String
+    - contextPath: Domain.Publications.timestamp
+      description: The timestamp in which the information was published.
+      type: Date
```
Comment (moishce) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:152]: My change

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 13]
```diff
@@ -160,20 +206,51 @@ script:
...
+      type: String
+    - contextPath: IP.Publications.title
+      description: The title of the published information.
+      type: String
+    - contextPath: IP.Publications.link
+      description: A link to the published information.
+      type: String
+    - contextPath: IP.Publications.timestamp
+      description: The timestamp in which the information was published.
+      type: Date
```
Comment (moishce) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:229]: My change

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 14]
```diff
@@ -220,23 +297,56 @@ script:
...
+    - contextPath: File.Publications.title
+      description: The title of the published information.
+      type: String
+    - contextPath: File.Publications.link
+      description: A link to the published information.
+      type: String
+    - contextPath: File.Publications.timestamp
+      description: The timestamp in which the information was published.
+      type: Date
+  - arguments:
```
Comment (moishce) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:321]: My change

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 15]
```diff
@@ -530,27 +654,67 @@ script:
...
+      type: String
+    - contextPath: URL.Publications.title
+      description: The title of the published information.
+      type: String
+    - contextPath: URL.Publications.link
+      description: A link to the published information.
+      type: String
+    - contextPath: URL.Publications.timestamp
+      description: The timestamp in which the information was published.
+      type: Date
```
Comment (moishce) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:677]: My change

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 16]
```diff
@@ -115,20 +129,51 @@ script:
...
       description: Risk last seen timestamp
       type: date
-    description: Returns threat intelligence information for a domain or DNS in Recorded Future.
-  - name: ip
-    arguments:
-    - name: ip
-      required: true
-      default: true
+    - contextPath: Domain.Tags
+      description: Tags that are associated with the Domain
```
Comment (roysagi) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:133]:
```suggestion
      description: Tags that are associated with the Domain.
```

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 17]
```diff
@@ -160,20 +206,51 @@ script:
     - contextPath: IP.RecordedFuture.LastSeen
       description: Risk last seen timestamp
       type: date
-    description: Returns threat intelligence information for an IP address in Recorded Future.
-  - name: file
-    arguments:
-    - name: file
+    - contextPath: IP.Tags
+      description: Tags that are associated with the Domain
```
Comment (roysagi) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:210]:
```suggestion
      description: Tags that are associated with the IP.
```

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 18]
```diff
@@ -220,23 +297,56 @@ script:
...
       description: Risk last seen timestamp
       type: date
-    description: Returns threat intelligence information for a file in Recorded Future.
-  - name: recorded-future-get-related-entities
-    arguments:
-    - name: entityType
-      required: true
-      auto: PREDEFINED
+    - contextPath: File.Tags
+      description: Tags that are associated with the Domain
```
Comment (roysagi) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:301]:
```suggestion
      description: Tags that are associated with the file.
```

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 19]
```diff
@@ -530,27 +654,67 @@ script:
     - contextPath: URL.RecordedFuture.LastSeen
       description: Risk last seen timestamp
       type: date
-    description: Returns threat intelligence information for a URL in Recorded Future.
-  - name: recorded-future-get-threats-hash
-    arguments:
-    - name: detailed
-      auto: PREDEFINED
+    - contextPath: URL.Tags
+      description: Tags that are associated with the Domain
```
Comment (roysagi) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:658]:
```suggestion
      description: Tags that are associated with the URL.
```

**Review by roysagi — COMMENTED** [review]
Please see my changes and fix in the readme as well.

**Review by ShahafBenYakir — APPROVED** [review]

File: Packs/Recorded_Future/ReleaseNotes/1_0_3.md [hunk 20]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Recorded Future
+- Added support for additional context outputs to the **ip**, **url**, **file** and **domain** commands.
```
Comment (amshamah419) [inline Packs/Recorded_Future/ReleaseNotes/1_0_3.md:4]:
```suggestion
- Added support for additional context outputs to the ***ip***, ***url***, ***file*** and ***domain*** commands.
```
